docs/technical/editor-integration.md

```diff
@@ -39,6 +39,7 @@ These editors are currently supported:
  - [JetBrains PyCharm](https://www.jetbrains.com/pycharm/)
  - [JetBrains RubyMine](https://www.jetbrains.com/rubymine/)
  - [JetBrains GoLand](https://www.jetbrains.com/go/)
+- [JetBrains RustRover](https://www.jetbrains.com/rust/)
```

Comment:
missing space here

Reply:
what ident？

Reply:
how i need indent？ i think this is normal indent👉👈

Reply:
ok i understood yours, thx!